source/swap/docs/Imports.md

```diff
@@ -0,0 +1,22 @@
+## <span id="Imports"></span> `Imports`
+
+``
```

Comment:
What are these for?

Reply:
Actually just confused by this whole doc. Why does Imports need a readme that explains authorizations?